wp-content/themes/engage/templates/page-about.twig

```diff
@@ -1,22 +1,29 @@
 {% extends "base.twig" %}
 
 {% block content %}
-	<article class="container container--md post-type--{{post.post_type}}" id="post--{{post.ID}}">
-		<header class="article__header">
-			<h1 class="article__title">{{post.title}}</h1>
-		</header>
-		<div class="article__content">
-			{{post.content}}
-		</div>
-		<div class="newsletter__wrapper">
-		  <div class="newsletter-page">
-		      {{ newsletter }}
-		  </div>
-		</div>
-		<div class="home-funders home-section">
-			<div class="container container--xl wrapper--pad">
-				{% include 'partial/funders.twig' with { funders: about.funders } %}
-			</div>
-		</div>
-	</article>
-{% endblock %}
+    <article class="container container--md post-type--{{post.post_type}}" id="post--{{post.ID}}">
+        <header class="article__header">
+            <h1 class="article__title">{{post.title}}</h1>
+        </header>
+    </article>
+    <article class="container container--lg post-type--{{post.post_type}}" id="post--{{post.ID}}">
+        <div class="about_row">
```

Comment:
I believe @Lay-ton already modified some things related to newsletters on about page. Can you two get together to make sure you're not repeating work?

Reply:
You're right I think I repeated some of his work, I talked to layton and i think we're going to be going with his version.